components/Chart.tsx

```diff
@@ -1,24 +1,36 @@
 import _ from "lodash"
-import Highcharts from "highcharts"
-import HighchartsAccessibility from "highcharts/modules/accessibility"
-import HighchartsReact from "highcharts-react-official"
 
 import { Pattern, PatternClass, Term } from "../lib/types"
 
-// https://github.com/highcharts/highcharts-react#highcharts-with-nextjs
-if (typeof Highcharts === "object") {
-  HighchartsAccessibility(Highcharts)
-}
-
 type Props = {
+  rollupToPatternClass: boolean,
   showLabels: boolean,
   terms?: Term[]
   patterns?: Pattern[]
   patternClasses?: PatternClass[]
 }
 
 const Chart = (props: Props) => {
-  const { showLabels, terms = [], patterns = [], patternClasses = [] } = props
+  const { rollupToPatternClass, showLabels, terms = [], patterns = [], patternClasses = [] } = props;
+
+  // Format the list of individual terms that apply to this entry
+  let formattedTerms = _(terms)
+    .map((term: any) => ({
+      pattern: _.find(patterns, ['_id', term.pattern._ref]),
+      patternName: _.find(patterns, ['_id', term.pattern._ref])?.name,
+      type: term.rightsIntensity > 0 ? "Right" : "Obligation", // because pattern.type is not consistently populated
```

Comment:
This is likely due to the staging / production dataset issues that I'll sort today and then take a close look 👍 

The other issue that would be in play here is that there are some null values for intensity meaning we cant derive a type - I'll see how many there are and see if I can get Niamh to prioritise populating these.

Reply:
we could also be a bit smarter here and do something like: `term.type || term.rightsIntensity > 0 ? "Right" : "Obligation"`

i think the side effect of not being able to derive a type is simply not rendering that row on the chart right now - so not ideal, but not breaking!